.drone.yml

```diff
@@ -69,6 +74,7 @@ pipeline:
     environment:
       TAGS: bindata sqlite sqlite_unlock_notify
     commands:
+      - git tag # To see if the tags were successfully pulled
```

Comment:
Will this be here permanently? Or do you want to remove it after the first build?

Reply:
Just removed it, was just to verify the tags are pulled correctly.

Reply:
Yes, please remove this before merge.